### ISSUE TITLE
Not able to get "delete" button in Portus 2.5

### ISSUE THREAD

**Issue by ashniku** [SUSE/Portus#2000]
### Description

Check out the `CONTRIBUTING.md` file for some considerations before submitting a
new issue.

### Steps to reproduce

```yml
version: "3"

services:
  portus:
    image: opensuse/portus:head
    environment:
      - PORTUS_MACHINE_FQDN_VALUE=${MACHINE_FQDN}
      - CCONFIG_PREFIX=PORTUS   # Added for Bug

      # DB. The password for the database should definitely not be here. You are
      # probably better off with Docker Swarm secrets.
      - PORTUS_DB_HOST=db
      - PORTUS_DB_DATABASE=portus_production
      - PORTUS_DB_PASSWORD=${DATABASE_PASSWORD}
      - PORTUS_DB_POOL=5

      # Secrets. It can possibly be handled better with Swarm's secrets.
      - PORTUS_SECRET_KEY_BASE=${SECRET_KEY_BASE}
      - PORTUS_KEY_PATH=/certificates/portus.key
      - PORTUS_PASSWORD=${PORTUS_PASSWORD}

      # SSL
      - PORTUS_PUMA_TLS_KEY=/certificates/portus.key
      - PORTUS_PUMA_TLS_CERT=/certificates/portus.crt

      - PORTUS_DELETE_ENABLED=true   # Added for delete functionalitya
      - PORTUS_DELETE_CONTRIBUTORS=true
      - PORTUS_DELETE_GARBAGE_COLLECTOR_ENABLED=true

      # NGinx is serving the assets instead of Puma. If you want to change this,
      # uncomment this line.
      #- RAILS_SERVE_STATIC_FILES='true'
    ports:
      - 3000:3000
    volumes:
      - ./secrets:/certificates:ro
      - static:/srv/Portus/public
    hostname: xxxxxxx.com
    networks:
      - portusNetwork
```

Should get delete button  
- **Actual behavior**: Not getting delete button for tags

I can see env variables are set:

  docker exec -it compose_portus_1_855c26cc2007  portusctl exec rake portus:info

delete:
  enabled: true
  contributors: true
  garbage_collector:
    enabled: true
    older_than: 30
    tag: ''
 Note: I have logged in with the admin user,that is created during configuration...

**Comment by mssola** [comment]
Could you send us some screenshot on what you actually see? I've tested this on current head, with delete enabled, and when I selected at least a tag, the button for deleting tags was shown :confused:

**Comment by mssola** [comment]
Also, could you tell me what `portusctl exec rake portus:info` prints if you execute it inside of the container?

**Comment by ashniku** [comment]
I will upload it soon..Thanks for the response. Applogies for the delay

**Comment by ashniku** [comment]
Please find the portus compose file:

version: "3.4"

services:
  portus:
    image: opensuse/portus:head
    environment:
      - PORTUS_MACHINE_FQDN_VALUE=${MACHINE_FQDN}
      - CCONFIG_PREFIX=PORTUS   # Added for Bug

      # DB. The password for the database should definitely not be here. You are
      # probably better off with Docker Swarm secrets.
      - PORTUS_DB_HOST=db
      - PORTUS_DB_DATABASE=portus_production
      - PORTUS_DB_PASSWORD=${DATABASE_PASSWORD}
      - PORTUS_DB_POOL=5

      # Secrets. It can possibly be handled better with Swarm's secrets.
      - PORTUS_SECRET_KEY_BASE=${SECRET_KEY_BASE}
      - PORTUS_KEY_PATH=/certificates/portus.key
      - PORTUS_PASSWORD=${PORTUS_PASSWORD}

      # SSL
      - PORTUS_PUMA_TLS_KEY=/certificates/portus.key
      - PORTUS_PUMA_TLS_CERT=/certificates/portus.crt

      - PORTUS_DELETE_ENABLED=true   # Added for delete functionality

      # NGinx is serving the assets instead of Puma. If you want to change this,
      # uncomment this line.
      #- RAILS_SERVE_STATIC_FILES='true'
    ports:
      - 3000:3000
    volumes:
      - ./secrets:/certificates:ro
      - static:/srv/Portus/public
    hostname: celvpvm09662.us.oracle.com
    networks:
      - portusNetwork

  background:
    image: opensuse/portus:head
    depends_on:
      - portus
      - db
    environment:
      # Theoretically not needed, but cconfig's been buggy on this...
      - CCONFIG_PREFIX=PORTUS
      - PORTUS_MACHINE_FQDN_VALUE=${MACHINE_FQDN}

      # DB. The password for the database should definitely not be here. You are
      # probably better off with Docker Swarm secrets.
      - PORTUS_DB_HOST=db
      - PORTUS_DB_DATABASE=portus_production
      - PORTUS_DB_PASSWORD=${DATABASE_PASSWORD}
      - PORTUS_DB_POOL=5

      # Secrets. It can possibly be handled better with Swarm's secrets.
      - PORTUS_SECRET_KEY_BASE=${SECRET_KEY_BASE}
      - PORTUS_KEY_PATH=/certificates/portus.key
      - PORTUS_PASSWORD=${PORTUS_PASSWORD}

      - PORTUS_BACKGROUND=true
    volumes:
      - ./secrets:/certificates:ro
    networks:
      - portusNetwork

  db:
    image: library/mariadb:10.0.23
    command: mysqld --character-set-server=utf8 --collation-server=utf8_unicode_ci --init-connect='SET NAMES UTF8;' --innodb-flush-log-at-trx-commit=0
    environment:
      - MYSQL_DATABASE=portus_production

      # Again, the password shouldn't be handled like this.
      - MYSQL_ROOT_PASSWORD=${DATABASE_PASSWORD}
    volumes:
      - /var/lib/portus/mariadb:/var/lib/mysql
    networks:
      - portusNetwork

  registry:
    image: library/registry:2.6
    command: ["/bin/sh", "/etc/docker/registry/init"]
    environment:
      # Authentication
      REGISTRY_AUTH_TOKEN_REALM: https://${MACHINE_FQDN}/v2/token
      REGISTRY_AUTH_TOKEN_SERVICE: ${MACHINE_FQDN}
      REGISTRY_AUTH_TOKEN_ISSUER: ${MACHINE_FQDN}
      REGISTRY_AUTH_TOKEN_ROOTCERTBUNDLE: /secrets/portus.crt

      # SSL
      REGISTRY_HTTP_TLS_CERTIFICATE: /secrets/portus.crt
      REGISTRY_HTTP_TLS_KEY: /secrets/portus.key

      # Portus endpoint
      REGISTRY_NOTIFICATIONS_ENDPOINTS: >
        - name: portus
          url: https://${MACHINE_FQDN}/v2/webhooks/events
          timeout: 2000ms
          threshold: 5
          backoff: 1s
    volumes:
      - /var/lib/portus/registry:/var/lib/registry
      - ./secrets:/secrets:ro
      - ./registry/config.yml:/etc/docker/registry/config.yml:ro
      - ./registry/init:/etc/docker/registry/init:ro
    ports:
      - 5000:5000
      - 5001:5001 # required to access debug service
    networks:
      - portusNetwork
    
  nginx:
    image: library/nginx:alpine
    volumes:
      - ./nginx/nginx.conf:/etc/nginx/nginx.conf:ro
      - ./secrets:/secrets:ro
      - static:/srv/Portus/public:ro
    ports:
      - 80:80
      - 443:443
    networks:
       #default:
        # aliases:
         # -  ${MACHINE_FQDN}
     - portusNetwork
    hostname: celvpvm09662.us.oracle.com


volumes:
  static:
    driver: local

networks:
  portusNetwork:
    driver: bridge

**Comment by ashniku** [comment]
portusctl exec rake portus:info
[Mailer config] Host:     xxxxxxxx
[Mailer config] Protocol: https://
Evaluated configuration:
---
email:
  from: portus@example.com
  name: Portus
  reply_to: ''
  smtp:
    enabled: false
    address: smtp.example.com
    port: 587
    domain: example.com
    ssl_tls: ''
    enable_starttls_auto: false
    openssl_verify_mode: none
    ca_path: ''
    ca_file: ''
    user_name: ''
    password: "****"
    authentication: login
gravatar:
  enabled: true
delete:
  enabled: true
  contributors: false
  garbage_collector:
    enabled: false
    older_than: 30
    tag: ''
ldap:
  enabled: false
  hostname: ldap_hostname
  port: 389
  timeout: 5
  encryption:
    method: ''
    options:
      ca_file: ''
      ssl_version: TLSv1_2
  base: ''
  filter: ''
  uid: uid
  authentication:
    enabled: false
    bind_dn: ''
    password: "****"
  guess_email:
    enabled: false
    attr: ''
oauth:
  local_login:
    enabled: true
  google_oauth2:
    enabled: false
    id: ''
    secret: ''
    domain: ''
    options:
      hd: ''
  open_id:
    enabled: false
    identifier: ''
    domain: ''
  github:
    enabled: false
    client_id: ''
    client_secret: ''
    organization: ''
    team: ''
    domain: ''
  gitlab:
    enabled: false
    application_id: ''
    secret: ''
    group: ''
    domain: ''
    server: ''
  bitbucket:
    enabled: false
    key: ''
    secret: ''
    domain: ''
    options:
      team: ''
first_user_admin:
  enabled: true
signup:
  enabled: true
check_ssl_usage:
  enabled: true
registry:
  jwt_expiration_time:
    value: 15
  catalog_page:
    value: 100
  timeout:
    value: 2
  read_timeout:
    value: 120
machine_fqdn:
  value: celvpvm09662.us.oracle.com
display_name:
  enabled: false
user_permission:
  change_visibility:
    enabled: true
  create_team:
    enabled: true
  manage_team:
    enabled: true
  create_namespace:
    enabled: true
  manage_namespace:
    enabled: true
  create_webhook:
    enabled: true
  manage_webhook:
    enabled: true
  push_images:
    policy: allow-teams
security:
  clair:
    server: ''
    health_port: 6061
    timeout: 900
  zypper:
    server: ''
  dummy:
    server: ''
anonymous_browsing:
  enabled: true
background:
  registry:
    enabled: true
  sync:
    enabled: true
    strategy: initial
pagination:
  limit: 10
  before_after: 2

Portus version: 2.5.0-dev@a562a2d53a1b7307631df1c5d0fef303e8491572

**Comment by ashniku** [comment]
![portuspng](https://user-images.githubusercontent.com/42708822/47655810-d2f99100-dbb3-11e8-9e53-9e0a3ae09f62.PNG)

**Comment by ashniku** [comment]
Can you please help me on this ?

**Comment by vitoravelino** [comment]
@ashniku There is no delete button in this page. Closing this but feel free to reopen as an enhancement to add a delete button in this page if you want.

**Comment by vitoravelino** [comment]
The delete button in the details page of the repository.

**Comment by ashniku** [comment]
Hi Vitor,

I dont see in the delete button. Can you plz check. applogies,if i am doing something naive.
![capture](https://user-images.githubusercontent.com/42708822/47664919-05ac8500-dbc6-11e8-8155-12e355da3c16.PNG)

**Comment by vitoravelino** [comment]
Ok. This is a bug and it's because the repository was pushed in the global namespace. Since I already closed this one, follow #2011 for new updates.
